src/pages/user_profile/ResearcherStatus.tsx

```diff
@@ -123,6 +125,22 @@ const ResearcherStatus: React.FC<ResearcherStatusProps> = (props) => {
       })}
       <div style={{ marginTop: '20px' }} />
       <p style={subheadStyle}>Library Card issued to you</p>
+      {signingOfficialUsers.length === 0
+        ? (
+            <p>
+              No Signing Official found for your institution. Please refer to <a href="https://duos.blog/2025/08/06/how-to-get-a-library-card-from-your-signing-official/" target="_blank" rel="noreferrer">this help article</a> for instructions on how to get a Library Card.
```

Comment:
Knowing referrer seems secure and potentially helpful here:

```suggestion
              No Signing Official found for your institution. Please refer to <a href="https://duos.blog/2025/08/06/how-to-get-a-library-card-from-your-signing-official/" target="_blank">this help article</a> for instructions on how to get a Library Card.
```

Reply:
ESLint: Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers: see https://mathiasbynens.github.io/rel-noopener/#recommendations (react/jsx-no-target-blank)